08_files_lab/02.py

```diff
@@ -35,29 +28,28 @@ def main():
     parser.add_argument("file_2", help="path to file")
     parser.add_argument("file_3", help="path to file")
     args = parser.parse_args()
-    if not is_valid_file_path(args.file_1) or not is_valid_file_path(args.file_2):
-        sys.exit()
+
     if os.path.isfile(args.file_3):
         print "Error - file",args.file_3,"alrady exist"
         sys.exit()
-    if not create_dictionaries(args.file_3):
-        print "Error - failed to create file dictionaries"
-        sys.exit()
-    
 
-    with open(args.file_1, "r") as src1:
-        with open(args.file_2, "r") as src2:
-            with open(args.file_3, "w") as dst:        
-                file1_lines = src1.readlines()
-                file2_lines = src2.readlines()
-                file3_lines = filter(None, sum(itertools.izip_longest(file1_lines, file2_lines), ()))
-                dst.writelines(file3_lines)
-                
-        
-        
+    try:
+        create_dictionaries(args.file_3)
 
-    
-           
+        file1_lines = load_file(args.file_1)
+        file2_lines = load_file(args.file_2)
+        with open(args.file_3, "w") as dst:
+            itr = itertools.izip_longest(file1_lines, file2_lines)

```

Comment:
Prefer: `line1, line2 = itertools.izip_longest(file1_lines, file2_lines)`
So you don't need the brackets